### DISCUSSION TITLE
Welcome to Cloudlog Discussions!

### DISCUSSION THREAD

**Discussion by magicbug** [magicbug/Cloudlog#736] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with users of Cloudlog. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Remember to use Issues for reporting bugs.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by magicbug** [comment]
I'm guessing most of you know who I am I started this crazy project all those years ago, I'm delighted that discussions are now available this means I will shut down the external forum and redirect everyone to here instead keeping everything in one place.

Pete, 2M0SQL

**Comment by M0TZO** [comment]
Not a coder by any standard, but encouraged Pete to join Patreon (_well finish the sign up_), and then signed up to support him first,. I've also got a few ideas into Cloudlog.

Happy to keep supporting the project.

Paul MØTZO

**Comment by dg9vh** [comment]
Hi,
it's me, Kim, user and submitter of some lines of code. User support for
some German users. Trying to make Cloudlog a bit more known in Germany and
trying to make the world a better place 😊

73 de Kim
DG9VH

Paul GIBSON <notifications@github.com> schrieb am Mi., 9. Dez. 2020, 01:25:

**Reply by magicbug** [reply]
Thanks for helping our German users! hopefully, 2021 will see full translation added 👍🏻

**Comment by Ciemon** [comment]
Another non-coder but am able to work on areas that just use text, such as
adding satellites and contributing to documentation.

I'd welcome some form of jobs list being used, not only to show a
development pathway but also somewhere to list non-coding aspects that need
work so that they can be picked up by others.

73
Ciemon
G0TRT

**Reply by magicbug** [reply]
The main thing that always needs work is the documentation which is a never-ending job! but @Ciemon  is one of the people who fixes my typos 👍🏻  

However happy to start a discussion on jobs that don't require code

**Comment by ghost** [comment]
Like a few other on here, I'm not a coder. I'm very much a user at the moment. I'm hoping that changed going forward, if only with documentation and such.

Thanks,
Dan M0TCB
